spec/registration_server_spec.rb

```diff
@@ -9,11 +9,13 @@
   let(:port) { 9000 }
   let(:http_server) { double("http_server") }
   let(:certs_factory) { double("certs factory") }
+  let(:updates_url) { URI("http://updates.suse.com/sles12") }
```

Comment:
AFAIK, yes, they're covered:

```
lib/aytests/image_builder.rb:      iso_path = URI(iso_url.to_s).host ? IsoRepo.get(iso_url) : iso_url
lib/aytests/iso_repo.rb:      iso_path = File.join(@dir, URI(uri).host, URI(uri).path)
spec/servlets/list_updates_spec.rb:  subject(:servlet) { AYTests::Servlets::ListUpdates.new(server, URI(updates_url)) }
spec/servlets/list_updates_spec.rb:  let(:updates_url) { URI("https://updates.suse.com/sles12") }
spec/registration_server_spec.rb:  let(:updates_url) { URI("http://updates.suse.com/sles12") }
```

But to make the dependencies clear, I've decided to require "uri" in Veewee definitions and also fix `list_updates_spec.rb` test.